rook-source-mysql/src/main/java/com/github/shyiko/rook/source/mysql/MySQLReplicationStream.java

```diff
@@ -65,6 +65,10 @@
 
     private volatile boolean groupEventsByTX = true;
 
+    private Set<Long> ignoredHostIds = new HashSet<Long>();
```

Comment:
@igrunskiy I would keep it aligned with MySQL terminology and call it `ignoredServerIds`.